wallet-new/src/Cardano/Wallet/Kernel/DB/BlockMeta.hs

```diff
@@ -63,6 +63,15 @@ instance Monoid AddressMeta where
 makeLenses ''AddressMeta
 deriveSafeCopy 1 'base ''AddressMeta
 
+instance SafeCopy (InDb AddressMeta) where
+    getCopy = contain $ do
+        isUsed <- safeGet
+        isChange <- safeGet
+        pure . InDb $ AddressMeta isUsed isChange
+    putCopy (InDb (AddressMeta isUsed isChange)) = contain $ do
+        safePut isUsed
+        safePut isChange
+
```

Comment:
This instance is superfluous, `InDb` should only be used to tag types defined in core, anything defined by the wallet itself (that we have control ovr) we can just use `deriveSafeCopy`.